pages/price-feeds/use-real-time-data/ton.mdx

```diff
@@ -124,6 +124,28 @@ This code snippet does the following:
 4. Prepares the update data and calculates the update fee.
 5. Updates the price feeds on the TON contract.
 
+When providing Pyth data to your contract, there are typically two main scenarios: either you call a method supplying TON, or you transfer jettons.
+
+They both are demonstrated in the [Pyth Connector example](https://github.com/pyth-network/pyth-examples/tree/main/price_feeds/ton/pyth-connector). 
+
+These same patterns are also used in the  
+[EVAA Protocol code](https://github.com/evaafi/contracts/tree/v8).
```

Comment:
Let's reference the code link in here for EVAA ?

`These same patterns` -  Can we specify the patterns above ? and then give example for each.